### PR TITLE
Draft for uri_for_route

### PR DESCRIPTION
**Pull request by xsawyerx**

### CODE REVIEW

**Comment by cromedome** [comment]
You seem to have a whole article from @GeekRuthie at the end of your article. You should prune it.

File: danceradvent/public/articles/2023/XX-navigating-the-dancefloor.pod [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+    $path = uri_for_route(
+        'req_form',
+        {
+            'id'    => 4,
+            'splat' => [ 'foo', 'bar', [ 'baz', 'quux' ] ],
+        },
+    );
+    # $path = /upload/request/4/foo/baz/baz/quux
+
+(Notice that when you're mixing these, the splat-liek arguments will
```
Comment (cromedome) [inline danceradvent/public/articles/2023/XX-navigating-the-dancefloor.pod:117]: `s/liek/like/;`

**Comment by cromedome** [comment]
I really like how you illustrate the problems a developer faces, and how this new keyword addresses the problem, making life better for Dancer developers. Excellent approach.

**Comment by xsawyerx** [comment]
> You seem to have a whole article from @GeekRuthie at the end of your article. You should prune it.

Bah. Had various crap there. Cleaned it up. Thanks!

**Comment by cromedome** [comment]
Thanks for going back to update the method calls in the templates. 👍

**Pull request merged**